tests/h/h_api/bulk_api/model/data_body_test.py

```diff
@@ -67,10 +68,25 @@ def test_create_ok(self):
             }
         }
 
-    def test_create_can_fail(self,):
-        with pytest.raises(Exception):
+    def test_create_can_fail(self):
+        with pytest.raises(SchemaValidationError):
             CreateGroupMembership.create("bad", None)
 
+    def test_validation_can_fail(self):
+        with pytest.raises(SchemaValidationError) as e:
+            CreateGroupMembership(
+                {
+                    "data": {
+                        "type": "group_membership",
+                        "relationships": {
+                            # This isn't good enough, these should have bodies
+                            "member": {},
+                            "group": {},
+                        },
+                    }
+                }
+            )
```

Comment:
I don't want to go crazy with tests like this, as the number of things that could be wrong is infinite, but this covers this particular case

Reply:
You could probably use `parametrize` to generate a large number of differently invalid requests and test them all as one test

Reply:
I was thinking about some kind of JSON fuzzer that would futz about with the data structure, but it seems over kill right now.